linkerd/app/outbound/src/ingress.rs

```diff
@@ -85,6 +110,8 @@ impl Outbound<()> {
             // inner service to readiness even if new requests aren't
             // received.
             .push_map_target(http::Logical::from)
+            .push(svc::UnwrapOr::layer(http_endpoint))
```

Comment:
Mmmm.. I don't think we want to support this? We've already dropped the origdstaddr from the target, right? So what endpoint can this possibly route to?

Reply:
Ah, yeah, I guess we should change this to a filter that rejects connections when there was no profile discovered?